includes/uninstall.php

```diff
@@ -0,0 +1,143 @@
+<?php
+/**
+ * Uninstall helpers.
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories;
+
+/**
+ * Deletes options and transients.
+ *
+ * @return void
+ */
+function delete_options() {
+	global $wpdb;
+
+	$prefix = 'web_stories\_%';
+
+	// phpcs:ignore WordPress.DB.DirectDatabaseQuery
+	$options = $wpdb->get_col(
+		$wpdb->prepare(
+			"SELECT option_name FROM $wpdb->options WHERE option_name LIKE %s",
+			$prefix
+		)
+	);
+
+	if ( ! empty( $options ) ) {
+		array_map( 'delete_option', (array) $options );
+	}
+
+
+	// phpcs:ignore WordPress.DB.DirectDatabaseQuery
+	$transients = $wpdb->get_col(
+		$wpdb->prepare(
+			"SELECT option_name FROM $wpdb->options WHERE option_name LIKE %s OR option_name LIKE %s",
+			'_transient_' . $prefix,
+			'_transient_timeout_' . $prefix
+		)
+	);
+
+	if ( ! empty( $transients ) ) {
+		array_map( 'delete_option', (array) $transients );
```

Comment:
Why doesn't this use `delete_transient` anymore?

If it's just using `delete_option`, we can just as well merge the two SQL queries.

Reply:
Because the name of the transient is prefixed. See 
https://github.com/WordPress/wordpress-develop/blob/5f6ab44340e7ecd4e99bbf5841e2f0689c4f8d1d/src/wp-includes/option.php#L649-L650

We get the prefixed name from the SQL query. I can remove the prefix or just use the `delete_option`. 

Worth noting, that only sites without object caching would have transient removed here. If object caching is enabled, then it stored stay in object cache and would return in this SQL query.  See https://github.com/WordPress/wordpress-develop/blob/5f6ab44340e7ecd4e99bbf5841e2f0689c4f8d1d/src/wp-includes/option.php#L646-L648

Reply:
Ah!

Then let's make just 1 SQL query for options & transients?

Reply:
We could do, I just didn't want to have 3 like searches in one query. It might make the query slow. Also, I think that a clearing transients is a little overkill. Lots of sites have object caching now, so the likely hood this query will return anything is low. Transients will just expire over time.

Reply:
Hmm I see. Let's keep it like this then. Perhaps we can remove it at some point.